script.js

```diff
@@ -0,0 +1,54 @@
+const header = document.querySelector('header');
+const div = document.createElement('div');
+const img = document.createElement('img');
+const ul = document.createElement('ul');
+let li = document.createElement('li');
+let a = document.createElement('a');
+
+function opened() {
+  div.style.display = 'block';
+}
+
+function closed() {
+  div.style.display = 'none';
+}
+
+div.className = 'extended';
+
+img.className = 'icon';
+img.src = './img/close.svg';
+div.appendChild(img);
+img.addEventListener('click', closed);
+
+ul.className = 'menu-extended';
+div.appendChild(ul);
+
+a.href = '#about';
```

Comment:
- [ ] I think there is a misunderstanding here you should make the project cards generated dynamically using `Javascript`
by storing projects information in an array and display that each project information and in more detail when the user click on the `See the project ` button

- project card
![image](https://user-images.githubusercontent.com/61790587/167504112-ee50562d-5b9e-4fa4-b997-57c0a190aee1.png)
- window popup for more detail
![image](https://user-images.githubusercontent.com/61790587/167504170-67fc4d48-f8ea-4aba-8e46-f68beca53d52.png)

and don't forget the design for mobile also